packages/components/src/ForumScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from 'react';
+import { Text } from 'react-native';
+import { useServices } from './services';
+import { ForumPosts } from './schemas/Forum.schema';
+
+const ForumScreen: React.FC = () => {
+  const [posts, setPosts] = useState<ForumPosts[]>([]);
+  const { forumService } = useServices();
+
+  const fetchPosts = async () => {
+    const data = await forumService.fetchPosts(1);
+    setPosts(data);
+  };
+
+  useEffect(() => {
+    fetchPosts();
+  }, []);
+
+  return <Text> {posts.length} </Text>;
+};
+
+export default ForumScreen;
```

Comment:
use named export instead of default export,

Reply:
aage jake hat jayegi ye screen, dummy hai for tetsing